documentation/design-docs/ipc-protocol.md

```diff
@@ -397,6 +336,59 @@ For example, the Command to start a stream session with EventPipe would be `0x02
 
 ## EventPipe Commands
 
+The EventPipe CommandSet enables Clients to create/start or stop [EventPipe](https://learn.microsoft.com/en-us/dotnet/core/diagnostics/eventpipe) Sessions. Two session types are currently supported through this protocol.
```

Comment:
A bunch of this info is specific to CollectTracing5. Putting it before all the commands suggests that it applies equally to all of them. I'd recommend moving this to the CollectTracing5 part of the doc.

Reply:
I was aiming for it to describe the latest/all of EventPipe commands capabilites and was even trying to figure out how to describe which parameter was introduced in which version, but I couldn't come up with a clean quick way to do so.

Done, merged this text with the text in CollectTracing5